src/components/App/App.js

```diff
@@ -9,7 +9,9 @@ export default class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: []
+      movies: [],
+      filteredMovies: [],
+      noFilteredMovies: false
```

Comment:
still don't love this, wish we could think of better naming. because we _technically_ start off with no filtered movies..